neon_homeassistant_skill/__init__.py

```diff
@@ -16,39 +17,112 @@ def chunks(lst, n_len) -> List[list]:
+        if self.connected:
+            self.enable_ha_intents()
+
+    def enable_ha_intents(self):
+        self.register_intents()
         self.bus.on("ovos.phal.plugin.homeassistant.assist.message.response", self._handle_assist_error)
         self.bus.on("ovos.phal.plugin.homeassistant.get.device.response", self.handle_get_device_response)
         self.bus.on("ovos.phal.plugin.homeassistant.device.turn_on.response", self.handle_turn_on_response)
         self.bus.on("ovos.phal.plugin.homeassistant.device.turn_off.response", self.handle_turn_off_response)
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.get.light.brightness.response",
-            self.handle_get_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.get.light.brightness.response",
+                self.handle_get_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.set.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.set.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.increase.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.increase.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.decrease.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.decrease.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.get.light.color.response",
-            self.handle_get_light_color_response,
-        )
+                "ovos.phal.plugin.homeassistant.get.light.color.response",
+                self.handle_get_light_color_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.set.light.color.response",
-            self.handle_set_light_color_response,
-        )
+                "ovos.phal.plugin.homeassistant.set.light.color.response",
+                self.handle_set_light_color_response,
+            )
+        self.bus.on("ovos.phal.plugin.homeassistant.ready", self.on_ready)
+
+    def on_ready(self, message):
+        resp = self.bus.wait_for_response(message.forward("ovos.phal.plugin.homeassistant.check_connected"))
+        self.log.debug(f"Response from HA PHAL plugin: {repr(resp)}")
+        if resp and resp.data.get("connected"):
+            self.log.debug("PHAL plugin connected to HA")
+            self.enable_ha_intents()
```

Comment:
Consider a check here if intents are already enabled that logs a warning to avoid errors upon re-registration